test/pkg/environment/aws/expectations.go

```diff
@@ -551,10 +551,14 @@ func ExpectCapacityReservationCreated(
 		}}, nil),
 	})
 	Expect(err).ToNot(HaveOccurred())
-	return lo.FromPtr(out.CapacityReservation.CapacityReservationId), func() {
-		GinkgoHelper()
+	return *out.CapacityReservation.CapacityReservationId
+}
+
+func ExpectCapacityReservationsCanceled(ctx context.Context, ec2api *ec2.Client, reservationIDs ...string) {
+	GinkgoHelper()
+	for _, id := range reservationIDs {
 		_, err := ec2api.CancelCapacityReservation(ctx, &ec2.CancelCapacityReservationInput{
-			CapacityReservationId: lo.ToPtr(*out.CapacityReservation.CapacityReservationId),
+			CapacityReservationId: &id,
 		})
 		Expect(err).ToNot(HaveOccurred())
```

Comment:
nit: Worth making this idempotent so it doesn't fail if it's not created or already canceled?